### PR TITLE
Fake Tweet Generator

### PR DESCRIPTION
**Pull request by JosephZheng1998**
Created the fake tweet generator that can generate fake tweets and usernames based on the existing tweets in the database. A detailed description can be found in README.md.

### CODE REVIEW

File: examples/twittermap/script/fake-tweet-generator/README.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+
+5. Run "text_generator.py" to get tweet ids from the AsterixDB database and pair them with fake texts. For ids associated with specified keywords, the script pairs them with texts generated for those keywords in "keyword_text_generator.py", which are stored in the json file. For other ids, "text_generator.py" generates random texts using the trained general model and pair each id with a fake text. Then the script inserts these {'id': <id>, 'text': <text>} records into the "ftext" dataset. It takes three arguments "-s" -- the url of the AsterixDB server, "-w" the keywords to generate tweets, and "-i" the filename storing the keyword json (e.g. local_keyword.json). There can be multiple "-w" flags, each followed by exactly one keyword. An example is "python3 text_generator.py -s http://localhost -w food -w world -w water -w happy -w dollar -i local_keyword.json". Please use "python3 text_generator.py --help" for a detailed description of the script and its arguments.
```
Comment (baiqiushi) [inline examples/twittermap/script/fake-tweet-generator/README.md]:
```suggestion
5. Run "text_generator.py" to get tweet ids from the AsterixDB database and pair them with fake texts. For ids associated with specified keywords, the script pairs them with texts generated for those keywords in "keyword_text_generator.py", which are stored in the json file. For other ids, "text_generator.py" generates random texts using the trained general model and pair each id with a fake text. Then the script inserts these {'id': <id>, 'text': <text>} records into the "ftext" dataset. It takes three arguments "-s" -- the url of the AsterixDB server, "-w" the keywords to generate tweets, and "-i" the filename storing the keyword json (which should be the output file of `keyword_text_generator.py`, e.g. local_keyword.json). There can be multiple "-w" flags, each followed by exactly one keyword. An example is "python3 text_generator.py -s http://localhost -w food -w world -w water -w happy -w dollar -i local_keyword.json". Please use "python3 text_generator.py --help" for a detailed description of the script and its arguments.
```

Comment (baiqiushi) [reply examples/twittermap/script/fake-tweet-generator/README.md]: I just added one sentence after `keyword json`

File: examples/twittermap/script/fake-tweet-generator/create_ds_tweet_coord.sql [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+
+
+drop dataset ds_tweet_coord if exists;
+
+create dataset ds_tweet_coord (typeTweet) if not exists primary key id;
+
+insert into ds_tweet_coord (
+select value d
+from ds_tweet d
+where d.coordinate is not unknown);
```
Comment (baiqiushi) [inline examples/twittermap/script/fake-tweet-generator/create_ds_tweet_coord.sql]:
```suggestion
where d.coordinate is not unknown or d.bounding_box is not unknown);
```

File: examples/twittermap/script/fake-tweet-generator/README.md (hunk 1 above)
Comment (JosephZheng1998) [reply examples/twittermap/script/fake-tweet-generator/README.md]: Cool

**Comment by baiqiushi** [comment]
Already moved to standalone project under ISG [AITweetGenerator](https://github.com/ISG-ICS/AITweetGenerator)

**Pull request closed without merging**